README.md

```diff
@@ -34,29 +34,25 @@
 ```jsx
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { createStore, combineReducers, applyMiddleware } from 'redux';
+import { createStore } from 'redux';
 import { createInputsReducer, connectWithInputs, ReduxInputsWrapper } from 'redux-inputs';
 import { Provider } from 'react-redux';
-import thunk = from 'redux-thunk';
 
 const inputsConfig = {
     email: {
         defaultValue: 'test@example.com',
         validator: (value) => typeof value === 'string' && value.indexOf('@') >= 0
     }
 };
-const reducer = combineReducers({
```

Comment:
I guess we do need to specify because redux inputs creates its own state at another mount point.